src/main/java/frc/robot/subsystems/Swerve/DrivebaseSubsystem.java

```diff
@@ -184,15 +190,29 @@ public Rotation2d getGyroscopeRotation() {
     public void periodic() {
 
         swerveOdometry.update(getGyroscopeRotation(), getModulePositions());
-
+        System.out.println(ll.getRobotPose());
```

Comment:
Strike this. This is the only place you use `ll` as well, so strike it too and do not create your broken `Limelight` subsystem. You're printing the same pose here every time... it comes from the single read of the limelight data in my other comment.